src/requirements.in

```diff
@@ -1,7 +1,8 @@
-django>=3.2.15, <4.0
+cryptography>=39.0.1
+django>=3.2.18, <4.0
 djangorestframework>=3.0, <4.0
 django-session-timeout>=0.1, <1.0
-docker-compose>=1.0, <2.0
+docker-compose>=1.0
```

Comment:
Why is `docker-compose` a requirement here? Shouldn't Docker and docker-compose be installed on the underlying Ubuntu system?

Reply:
Docker must be installed through the underlying system. However, it looks like docker-compose can either be installed through the system or through pip. See documentation here:
https://pypi.org/project/docker-compose/

I am fine with removing it if you think we should, should I remove?

Reply:
I see- let's leave this. I think it is convenient to have and removes a manual installation step. It also lets us control the version independently of what might be installed on the host.

Docker-compose v2 is completely rewritten from v1, so we should keep the <2.0 version ceiling here.

Reply:
Looking more closely, the `docker-compose` PyPi package seems to only support Docker Compose V1. Docker Compose V2 is a rewrite in Golang instead of Python, and it appears the Docker team does not intend to make it installable via Python. No version ceiling is OK here.